### PR TITLE
Update k8s version

### PR DESCRIPTION
**Pull request by aslom**
For 0.21 supported minimum version is 0.18 according to https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md#knative-eventing-version-table
Adding link to list of versions 
@lionelvillard

"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/install/any-kubernetes-cluster.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ This guide assumes that you want to install an upstream Knative release on a
 Kubernetes cluster. A growing number of vendors have managed Knative offerings;
 see the [Knative Offerings](../knative-offerings.md) page for a full list.
 
-Knative {{< version >}} requires a Kubernetes cluster v1.17 or newer, as well as
+Knative {{< version >}} requires a Kubernetes cluster [v1.18 or newer](https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md#knative-eventing-version-table), as well as
```
Comment (lionelvillard) [inline docs/install/any-kubernetes-cluster.md:44]: The link should be https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md#knative-version-tables

Comment (aslom) [reply docs/install/any-kubernetes-cluster.md]: Fixed

**Review by abrennan89 — APPROVED** [review]
This should probably also be added to the main branch if it applies for 0.21+, and CP'd to 0.22?

**Comment by abrennan89** [comment]
Ah, nevermind, the docs were updated for 0.22+ so this doesn't apply.

**Pull request merged**